packages/Uploader/src/components/File/File.styles.js

```diff
@@ -12,6 +12,10 @@ export const File = styled.li`
   &:last-of-type {
     border-bottom: 0;
   }
+
```

Comment:
Little bit of scope creep, but can we do the following on line `8` above, please? 🙏 

```css
border-bottom: 1px solid ${tokens.border.color};
```